src/main/kotlin/no/nav/syfo/fastlege/api/FastlegeAPI.kt

```diff
@@ -0,0 +1,33 @@
+package no.nav.syfo.fastlege.api
+
+import io.ktor.application.*
+import io.ktor.response.*
+import io.ktor.routing.*
+import no.nav.syfo.fastlege.ws.adresseregister.AdresseregisterClient
+import no.nav.syfo.fastlege.ws.fastlege.FastlegeInformasjonClient
+import no.nav.syfo.util.*
+
+const val fastlegeBasePath = "/api/v1/fastlege"
+const val fnrParam = "fnr"
+
+fun Route.registerFastlegeApi(
+    fastlegeClient: FastlegeInformasjonClient,
+    adresseregisterClient: AdresseregisterClient,
+) {
+    route(fastlegeBasePath) {
+        get("/{$fnrParam}") {
```

Comment:
Fint om vi kan oppgradere til å ta i mot personIdent som header her i stedet for path-param som er brukt tidligere(slik som i isdialogmote etc.)

Reply:
Ja, det høres bedre ut.